content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -38,18 +42,59 @@ and
     +-org.slf4j:jcl-over-slf4j:1.7.26
 ----
 
+This error says that a conflict exists between the versions of `commons-codec` and `jcl-over-slf4j` specified by the plugin versus Jenkins.
+See
+link:https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes[Understanding requireUpperBoundDeps failures and fixes] for more information.
+
+This page details ways to simplify the management of dependency versions.
+
+== Apache Maven Bill of Materials (BOM)
 
-What this error is saying is that there is a conflict between the versions of `commons-codec` and `jcl-over-slf4j` specified by the plugin versus Jenkins.
-Without using the BOM, you would need to go and update the dependencies in your plugin to match those required by Jenkins, and then keep changing these as your `jenkins.version` changes.
-With the BOM all you have to do is import the link:https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml[jenkins-bom] for the version of Jenkins you are building against, and the versions of these and other common dependencies will be matched to that version of Jenkins.
+The link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Apache Maven Bill Of Materials (BOM)]
+defines the versions of various libraries used by Jenkins.
+It is included in version 2.195 (and LTS version 2.164.x) and later of Jenkins Core.
 
-To use the Jenkins Core BOM in your plugin, just use plugin-pom (4.0 or later).
+If you use Maven to build your plugin,
+dependencies are defined in the `dependencyManagement` section of you _pom.xml_ file.
+You can import the appropriate Jenkins BOM to your _pom.xml_ file
+so that dependency versions are automatically synchronised
+with the version of Jenkins against which you are building.
 
-== Jenkins Plugin BOM
+To use the Jenkins Core BOM in your plugin:
+
+////
+Does this need a hint about how to be sure one is using this?
+////
+. Be sure that you are using
+link:https://github.com/jenkinsci/plugin-pom/[Parent POM for Jenkins Plugins] 
+(4.0 or later).
+. Go to the the link:https://github.com/jenkinsci/bom[jenkinsci/bom] repository.
```

Comment:
This section isn't right, there's two bom's.

1. Core bom
2. Plugin bom

Core bom exposes the libraries used by core
Plugin bom is plugins that have been tested to work together

The docs were right before when it said:
> To use the Jenkins Core BOM in your plugin, just use plugin-pom (4.0 or later).

Reply:
So just refer to plugin-pom with no link?  Is this always installed?

Reply:
I've added some additional text and a video tutorial for
* Update parent pom to use core BOM
* Add plugin BOM to simplify dependency management
* Add Dependabot to automate dependency management